tools/tpm2_getmanufec.c

```diff
@@ -277,7 +277,7 @@ char *Base64Encode(const unsigned char* buffer)
     bio = BIO_push(b64, bio);
     BIO_set_flags(bio, BIO_FLAGS_BASE64_NO_NL);
     BIO_write(bio, buffer, SHA256_DIGEST_LENGTH);
-    BIO_flush(bio);
+    (void)BIO_flush(bio);
```

Comment:
Don't just ignore it, handle it

Reply:
I did a `grep -R BIO_flush` on openssl itself. 99% of invocations ignore the return code.
According to the man page, we'd have to call `BIO_should_retry` and if true, "try again at a later time".
If `BIO_should_retry` is false though, I wouldn't even know what to do ?!?

So should I really make this a
```
while ((r = BIO_flush(bio)) != 1 && BIO_shoud_retry(bio)) {
    sleep(1);
}
if (r) {
     LOG_ERR("Flush failed");
}
```
Sounds awful... ;-)
IMHO, I'd stick with `(void)`. But I'm no openssl-expert...

Reply:
Use our UNUSED(x) macro.

Reply:
That I can do...